manifest.json

```
@@ -34,7 +34,7 @@
     "vtex.shipping-estimate-translator": "2.x",
     "vtex.format-currency": "0.x",
     "vtex.store-graphql": "2.x",
-    "vtex.apps-graphql": "2.x",
+    "vtex.apps-graphql": "3.x",
     "vtex.session-client": "1.x"
   },
   "settingsSchema": {
```

It'll be necessary to add the access public property
```suggestion
  "settingsSchema": {
  "access": "public",
```